### PR TITLE
added softmax_act param

### PR DESCRIPTION
**Pull request by Wickstrom**
Pull request for Issue #98  (Include softmax_act in model.predict in params of metrics). Added softmax_act as a parameter in all metrics that previously had the softmax_act specified. All metrics have the same default value for softmax_act as prior to this PR.

### CODE REVIEW

**Review by annahedstroem — CHANGES_REQUESTED** [review]
Thanks @Wickstrom for adding this.

All looks good but 2 tiny, tiny things:
1. Can we simply rename `softmax_act` to `softmax` (`act` stands for activation function but I find it redundant)?
2. Can we keep list namings as`pred_deltas` as is instead of `logits_deltas` (since `pred` stands for predictions and not probabilities let's keep it as is).

**Comment by Wickstrom** [comment]
1. Good point, I agree. Will update it.
2. I might be be misunderstanding something, but just so we are on the same page, you want the lists to be named pred_deltas? In most of the metrics that is the name, I think it was only in the faithfulness estimate that the name was logits_deltas, which I changed to pred_deltas.

**Comment by annahedstroem** [comment]
All good. Thanks @Wickstrom
